mob.go

```diff
@@ -1211,6 +1226,16 @@ func start(configuration Configuration) error {
 	return nil // no error
 }
 
+func createRemoteBranch(configuration Configuration, currentBaseBranch Branch) {
+	if !currentBaseBranch.hasRemoteBranch(configuration) && configuration.StartCreate {
+		git("push", configuration.RemoteName, currentBaseBranch.String(), "--set-upstream")
+	}
+
+	if currentBaseBranch.hasRemoteBranch(configuration) && configuration.StartCreate {
+		debugInfo("Remote branch " + currentBaseBranch.remote(configuration).String() + " already exists")
+	}
```

Comment:
The first if creates the remote branch and the second if then detect that newly created remote branch.  Is that expected behavior?

Reply:
Good catch, I think this needs an else. Also we could consider making the debugInfo an info

Reply:
Very good catch :) added it to the tests and fixed it :)